services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -104,11 +88,7 @@ export const ChoiceListField = ({
     });
   };
 
-  const autosaveContainsParentChoice = (value: string) =>
-    lastAutosaveValue &&
-    lastAutosaveValue.some((autosave) => autosave.value === value);
-
-  const clearChildren = (choices: FieldChoice[], dataToSend?: AnyObject) => {
```

Comment:
Note: I simplified this a bit because I don't think we'll need all the functionality with the new `getNestedFields` function.